app/Database/CourseQueries.hs

```diff
@@ -271,3 +273,22 @@ queryGraphs =
         do graphs :: [Entity Graph] <- selectList [] []
            return $ createJSONResponse graphs
 
+-- ========================================================
+
+getLectureTime :: [String] -> IO (Maybe [Time])
+getLectureTime [code, section, session] = runSqlite databasePath $ do 
```

Comment:
So I don't need to handle wrong cases, just let complier check and give error msg in getActiveTimetable?